### DISCUSSION TITLE
Don't want to support ie11 and older browser with polyfill

### DISCUSSION THREAD

**Discussion by wenerme** [vercel/next.js#11722] [Help] [answered]
Is there any option that make the compiled script only support for modern browsers ? like latest 3 version of google, es2016+, https://kangax.github.io/compat-table/es2016plus/#chrome79

**Comment by Timer** [comment, marked as answer]
The latest version of Next.js already does not polyfill for IE11 and older browsers by default. The code is only loaded in legacy browsers and does not slow down compilation.

**Reply by wenerme** [reply]
Thanks!
